README.md

```diff
@@ -13,13 +12,14 @@ https://github.com/mfuezesi/docker-wordmove), with WP-CLI support added.
 * mysql-client-5.5
 * php5
 * wp-cli
-* ENV RUBYOPT="-KU -E utf-8:utf-8" (Fix for some mysql sync issues)
+* ENV RUBYOPT="-KU -E utf-8:utf-8" (Fix for some mysql sync issues when using old
+  db adapter)
 
 ## How to use
 
 ### To run this image
 
-`docker run -it --rm -v ~/.ssh:/home/wordmove/.ssh:ro welaika/wordmove bash`
```

Comment:
Simon, am I wrong thinking that `bash` at the end of the command should be already launched by the `CMD` instruction in the `Dockerfile`?